packages/terra-alert/tests/jest/Alert.test.jsx

```diff
@@ -115,6 +118,125 @@ describe('Dismissable Alert of type custom with action button, custom title and
   });
 });
 
+describe('Dismissible Alert', () => {
+  let wrapper;
+
+  describe('Custom Alert with no title', () => {
+    beforeEach(() => {
+      wrapper = shallowWithIntl(
+        <Alert
+          type={Alert.Opts.Types.CUSTOM}
+          onDismiss={() => { }}
+          customIcon={<IconHelp />}
+        >
+          This is a custom alert.
+        </Alert>,
+      ).dive();
+    });
+    it('should set the alert message ID', () => {
+      const alertContent = wrapper.find('.section');
+      expect(alertContent.prop('id')).toEqual(`alert-message-${mockUUID}`);
+    });
+
+    it('should set the dismiss button aria-describedby to the alert description', () => {
+      const dismissButton = wrapper.find('Button');
+      expect(dismissButton.prop('aria-describedby')).toEqual(`alert-message-${mockUUID}`);
+    });
+  });
+
+  describe('Custom Alert with custom title', () => {
+    beforeEach(() => {
+      wrapper = shallowWithIntl(
+        <Alert
+          type={Alert.Opts.Types.CUSTOM}
+          title="Help!"
+          onDismiss={() => { }}
+          customIcon={<IconHelp />}
+        >
+          This is a custom alert.
+        </Alert>,
+      ).dive();
+    });
+
+    it('should set the alert message ID', () => {
+      const alertContent = wrapper.find('.section');
+      expect(alertContent.prop('id')).toEqual(`alert-message-${mockUUID}`);
+    });
+
+    it('should set the alert title ID', () => {
+      const alertTitle = wrapper.find('.title');
+      expect(alertTitle.prop('id')).toEqual(`alert-title-${mockUUID}`);
+    });
+
+    it('should set the dismiss button aria-describedby to the alert title', () => {
+      const dismissButton = wrapper.find('Button');
+      expect(dismissButton.prop('aria-describedby')).toEqual(`alert-title-${mockUUID}`);
+    });
+  });
+
+  describe('Success Alert with no title', () => {
+    beforeEach(() => {
+      wrapper = shallowWithIntl(
+        <Alert
+          type={Alert.Opts.Types.SUCCESS}
+          onDismiss={() => { }}
+        >
+          This is a success alert.
+        </Alert>,
+      ).dive();
+    });
+
+    it('should set the alert message ID', () => {
+      const alertContent = wrapper.find('.section');
+      expect(alertContent.prop('id')).toEqual(`alert-message-${mockUUID}`);
+    });
+
+    it('should set the alert title ID', () => {
+      const alertTitle = wrapper.find('.title');
+      expect(alertTitle.prop('id')).toEqual(`alert-title-${mockUUID}`);
+    });
+
+    it('should set the dismiss button aria-describedby to the alert title', () => {
+      const dismissButton = wrapper.find('Button');
+      expect(dismissButton.prop('aria-describedby')).toEqual(`alert-title-${mockUUID}`);
```

Comment:
Just confirming after doing a double take, since this is a success alert with no title, but success has default title of "Success" then the described by will mention the "Success" title?

Reply:
edit: completely overlooked your custom alert unit test! +1

Reply:
Yes, this test is for a Success alert with no title  prop passed in, but the alert will still display the default "Success" title. Updated the wording for those tests here: [faf94f6](https://github.com/cerner/terra-core/pull/3791/commits/faf94f693963e99e57103da845006f3254d0f5c3)